ui/lib/core/addon/components/enable-input.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface Args {
+  attr?: AttrData;
+}
+interface AttrData {
+  name: string; // required if @attr is passed
+  options?: {
+    label?: string;
+    helpText?: string;
+    subText?: string;
+    possibleValues?: string[];
+  };
+}
```

Comment:
I considered that, but decided against it because `ReadonlyFormField` doesn't support all of the same options that `FormField` does. I thought it might be best to keep this simple for what's relevant to this component and have a separate, more robust declaration for all of the options that can be passed to the expand attributes method. What do you think?

Reply:
Yes that sounds good to me!